lib/google_crawler/search/search_keyword_worker.ex

```diff
@@ -0,0 +1,84 @@
+defmodule GoogleCrawler.SearchKeywordWorker do
```

Comment:
But if this process dies then all state would be gone, right? 

When restarting should it then reads from the database to pick up all keyword remaining to be scraped? 🤔

Reply:
Yep true. I haven't think about this case 😂